frontend/packages/ceph-storage-plugin/src/components/ocs-install/ocs-request-data.ts

```diff
@@ -0,0 +1,50 @@
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { NO_PROVISIONER } from '../../constants';
+
+export const getOCSRequestData = (
+  scName?: string,
+  storage?: string,
+  infra?: string,
+): K8sResourceKind => {
+  const requestData = {
+    apiVersion: 'ocs.openshift.io/v1',
+    kind: 'StorageCluster',
+    metadata: {
+      name: 'ocs-storagecluster',
+      namespace: 'openshift-storage',
```

Comment:
I might not recall right, but there is an existing constant for ns.
```suggestion
      namespace: CEPH_STORAGE_NAMESPACE,
```